release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -284,6 +284,182 @@ To support this feature, OLM now enforces CRD upgrades are safer by ensuring
+    spec:
+      containers:
+      - args:
+...
+        - --feature-gates
+        - OperatorLifecycleManagerV2=true
+----
+
+.. Save your changes.
+
+. Install an Operator using the normal OperatorHub method if you have not already;
+this example uses an etcd Operator installed in the project `test-project`.
+
+. Create a new Operator resource for the installed etcd Operator.
+
+.. Save the following to a file:
++
+.`etcd-test-op.yaml` file
+----
+apiVersion: operators.coreos.com/v2alpha1
+kind: Operator
+metadata:
+  name: etcd-test
+----
+
+.. Create the resource:
++
+----
+$ oc create -f etcd-test-op.yaml
+----
+
+. To have the installed Operator opt in to the new API, apply the
+`operators.coreos.com/etcd-test` label to the following objects related to your
+Operator:
++
+--
+* Subscription
+* InstallPlan
+* ClusterServiceVersion
+* Any CRDs owned by the Operator
+--
++
+[NOTE]
+====
+In a future release, these objects will be automatically labeled for any
+Operators where the CSV was installed using a Subscription.
+====
++
+For example:
++
+----
+$ oc label sub etcd operators.coreos.com/etcd-test="" -n test-project
+$ oc label ip install-6c5mr operators.coreos.com/etcd-test="" -n test-project
+$ oc label csv etcdoperator.v0.9.4 operators.coreos.com/etcd-test="" -n test-project
+$ oc label crd etcdclusters.etcd.database.coreos.com operators.coreos.com/etcd-test="" -n test-project
+$ oc label crd etcdbackups.etcd.database.coreos.com operators.coreos.com/etcd-test="" -n test-project
+$ oc label crd etcdrestores.etcd.database.coreos.com operators.coreos.com/etcd-test="" -n test-project
+----
+
+. Verify your Operator has opted in to the new API.
+
+.. List all `operators` resources:
++
+----
+$ oc get operators --all-namespaces
+
+NAME        AGE
+etcd-test   17m
+----
+
+.. Inspect your Operator's details and note that the objects you labeled are
+represented:
++
+----
+$ oc describe operators etcd-test
+
+Name:         etcd-test
+Namespace:
+Labels:       <none>
+Annotations:  <none>
+API Version:  operators.coreos.com/v2alpha1
+Kind:         Operator
+Metadata:
+  Creation Timestamp:  2020-06-30T16:02:59Z
+  Generation:          1
+  Resource Version:    31411
+  Self Link:           /apis/operators.coreos.com/v2alpha1/operators/etcd-test
+  UID:                 85784d0f-7a2f-407a-96d3-938296d8f8b2
+Status:
+  Components:
+    Label Selector:
+      Match Expressions:
+        Key:       operators.coreos.com/etcd-test
+        Operator:  Exists
+    Refs:
+...
+      Kind:                    CustomResourceDefinition
+      Name:                    etcdclusters.etcd.database.coreos.com
+      API Version:             operators.coreos.com/v1alpha1
+      Conditions:
+        Last Transition Time:  2020-06-30T16:06:45Z
+        Message:               all available catalogsources are healthy
+        Reason:                AllCatalogSourcesHealthy
+        Status:                False
+        Type:                  CatalogSourcesUnhealthy
```

Comment:
This looks like the condition for a `CatalogSource` CR and not a CRD (which is the Kind in this example ref).

Reply:
Comparing the output from `oc get operator -o yaml` vs `oc describe operator`, the dashes with `oc get` does make it look more obvious which Kind the Conditions list is associated with, whereas `oc describe` looks more flat and you have to pay a bit more attention that each new object starts at the `API Version`:

`oc get` - https://gist.github.com/adellape/f4db67119d0b2ac0f8d27838a2381364
`oc describe` - https://gist.github.com/adellape/b77976b82e38f56e1c677428667dd916

I've modified the output in the doc to hopefully be more clear, and added some callouts to highlight where each object is named.

Reply:
OK to take "oc describer ...", thanks